inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -178,13 +179,27 @@ ie::CNNNetwork FrontEnd::convertNetwork(ie::CNNNetwork& network) {
         ngraph::opset5::Convolution::type_info,
         ngraph::opset5::GroupConvolution::type_info
     });
-
+    const std::vector<std::pair<ngraph::element::Type, ngraph::element::Type>> convert_precision_list {
+        {ngraph::element::i64, ngraph::element::i32},
+        {ngraph::element::u64, ngraph::element::i32},
+        {ngraph::element::u32, ngraph::element::i32},
+        {ngraph::element::u16, ngraph::element::i32},
+        {ngraph::element::boolean, ngraph::element::i32},
+    };
+    for (const auto &precision : convert_precision_list) {
+        manager.register_pass<ngraph::pass::ConvertPrecision>(precision.first, precision.second, myriad_type_to_fuse);
+    }
     manager.register_pass<vpu::DynamicToStaticShape>();
```

Comment:
`DynamicToStaticShape` pass inserts all of `StaticShape*` operations, But you call `ConvertPrecision` passes before `DynamicToStaticShape`. Is it reasonable?

Reply:
If we don't do it here, we get the following error:
``` 
[ ERROR ] Check 'get_element_type().is_dynamic() || get_element_type() == element_type' failed at core/src/runtime/host_tensor.cpp:188:
Can not change a static element type
```
It happens when we run the `ConstantFolding` pass we get `int32` in `get_element_type()` function type when `bool` is expected.

Reply:
Looks strange as far there were no such conversion before this commit and no error occurred

Reply:
@Maxim-Doronin @KorA6 

I think we should discuss this PR offline